jenkins/pipelines/ci/tiflash/tiflash-build-common.groovy

```diff
@@ -130,7 +130,12 @@ def getBuildTarget() {
     }
 
     if (params.BUILD_TESTS) {
-        targets = "bench_dbms gtests_dbms gtests_libcommon gtests_libdaemon ${targets}"
+        targets = "gtests_dbms gtests_libcommon gtests_libdaemon ${targets}"
+
+        // Only build bench binary for the master branch
+        if (params.TARGET_BRANCH == "master") {
```

Comment:
Should `release-7.5` be the same?

Reply:
I think only make sure the `bench_dbms` is not broken on the master branch by new PRs is OK. We have no requirement to run it on the release branch now.